sonar-ws/src/main/protobuf/ws-components.proto

```diff
@@ -57,6 +57,12 @@ message BulkUpdateKeyWsResponse {
   }
 }
 
+// WS api/components/search_projects
+message SearchProjectsWsResponse {
+  optional sonarqube.ws.commons.Paging paging = 1;
+  repeated Component components = 2;
```

Comment:
It's ok for the moment, but when we'll add measures it won't return this 'Component' object, am I right ?